edk2toolext/nuget_publishing.py

```diff
@@ -207,15 +207,20 @@ def _MakeNuspecXml(self, ContentDir, ReleaseNotesText=None):
     def _GetNuPkgFileName(self, version):
         # Nuget removes leading zeros so to match we must do the same
         s = self.Name + "."
+        append_tag = None
         parts = version.split(".")
-        for a in parts:
-            s += str(int(a)) + "."
+        if "-" in parts[-1]:
+            parts[-1], append_tag = parts[-1].split("-")
```

Comment:
If you're using this multiple assignment here...

Reply:
Fair